### ISSUE TITLE
NTR: Nerve conduction: F-waves reduced or absent (Lab: neurophysiology: NCV/EMG)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9052]
New term request Nerve conduction: F-waves reduced or absent (connol_101116160001):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000150
- label: Reduced F-wave amplitude
- parents: [Abnormal peripheral action potential amplitude (HP:0030179)](https://hpo.jax.org/app/browse/term/HP:0030179)
- pmid: PMID:17334607
- def: Reduced strength (amplitude) of the F-wave, which is an electrophysiological artifact produced by antidromic activation of motoneurons following distal electrical stimulation of motor nerve fibers.
- comment: Because F-waves traverse the entire length of a peripheral nerve between the spinal cord and muscle, F-waves provide a means of examining transmission between stimulation sites in the arm and the leg and the related motoneurons in the cervical and lumbosacral cord. The F-wave is so named because it was originally studied in the small muscles of the foot. It is one of several responses that may follow the direct motor (M) response evoked by electrical stimulation of mixed or motor nerves.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9052
- synonyms:
